react/modules/tasks/nextjs-ssr-ssg.md

```diff
@@ -2,39 +2,39 @@
 
 ## Technical requirements
 
-1. Create a separate branch for this task from the previous branch task. Branch name: "nextjs-ssr-pages-api".
-2. Next.js Pages Api Integration:
-   - [Migrate your application from vite to next.js by using Pages folder API](https://www.geeksforgeeks.org/next-js-upgrading-migrating-from-vite/?ysclid=m7dms13nb6414592988#steps-to-migrate-from-vite).
-   - React-router should be removed. You must use file-based routing provided by next.js (Pages Api).
-   - You may need to adapt some existing libraries to work with next.js if necessary.
-3. Next.js App Router Api Integration:
-   - Create a separate branch for this point from "nextjs-ssr-pages-api" branch. Branch name: "nextjs-ssr-app-router-api".
-   - You must use file-based routing provided by next.js (App Router)
-   - [Migrate from Pages Api to App Router](https://nextjs.org/docs/pages/building-your-application/upgrading/app-router-migration).
-4. [Optional] React Router 7 Integration:
-   - Create a separate branch for this point from the previous branch task. Branch name: "react-router-ssr".
-   - [Update routing](https://reactrouter.com/start/framework/routing)
-   - Migrate your application to SSR with React Router 7: data fetching should be done via loaders on the server-side
+1. Create a separate branch for this task from the previous branch task. Branch name: "nextjs-ssr".
+2. Next.js App Router Integration:
+   - [Migrate your application from vite to the Next.js by using App Router](https://nextjs.org/docs/app/guides/migrating/from-vite).
+   - React-router should be removed. You must use file-based routing provided by next.js (App Router).
+   - You may need to adapt some existing libraries to work with next.js if necessary. (Redux (can be replaced with [zustand](https://zustand.docs.pmnd.rs/guides/nextjs))).
```

Comment:
I suggest removing nested parenthesies, also Zustand will be an option together with Redux in the State management module, so the statement might be confusing.

Reply:
Fixed